backend/danswer/utils/variable_functionality.py

```diff
@@ -16,6 +17,7 @@ def get_is_ee_version(self) -> bool:
 global_version = DanswerVersion()
 
 
+@functools.lru_cache(maxsize=128)
```

Comment:
this could and should probably not have a maxsize, but just being safe to not OOM the container